how-to-guides/reproduce-run/notebooks/reproduce_run.ipynb

```diff
@@ -0,0 +1,435 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import torch.nn as nn\n",
+    "\n",
+    "\n",
+    "class BaseModel(nn.Module):\n",
+    "    def __init__(self, input_sz, hidden_dim, n_classes):\n",
+    "        super(BaseModel, self).__init__()\n",
+    "        self.main = nn.Sequential(\n",
+    "            nn.Linear(input_sz, hidden_dim * 2),\n",
+    "            nn.ReLU(),\n",
+    "            nn.Linear(hidden_dim * 2, hidden_dim),\n",
+    "            nn.ReLU(),\n",
+    "            nn.Linear(hidden_dim, hidden_dim // 2),\n",
+    "            nn.ReLU(),\n",
+    "            nn.Linear(hidden_dim // 2, n_classes),\n",
+    "        )\n",
+    "\n",
+    "    def forward(self, input):\n",
+    "        x = input.view(-1, 32 * 32 * 3)\n",
+    "        return self.main(x)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = BaseModel(\n",
+    "    old_run_params[\"input_sz\"],\n",
+    "    old_run_params[\"input_sz\"],\n",
+    "    old_run_params[\"n_classes\"],\n",
+    ")\n",
+    "criterion = nn.CrossEntropyLoss()\n",
+    "optimizer = torch.optim.SGD(model.parameters(), lr=old_run_params[\"lr\"])"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 5: Log losses and metrics"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "for i, (x, y) in enumerate(trainloader, 0):\n",
+    "    optimizer.zero_grad()\n",
+    "    outputs = model.forward(x)\n",
+    "    _, preds = torch.max(outputs, 1)\n",
+    "    loss = criterion(outputs, y)\n",
+    "    acc = (torch.sum(preds == y.data)) / len(x)\n",
+    "\n",
+    "    new_run[\"training/batch/loss\"].append(loss)\n",
+    "\n",
+    "    new_run[\"training/batch/acc\"].append(acc)\n",
+    "\n",
+    "    loss.backward()\n",
+    "    optimizer.step()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Stop logging\n",
+    "\n",
+    "Once you are done logging, stop tracking the run."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "old_run.stop()\n",
+    "new_run.stop()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Conclusion\n",
+    "You learned how to:\n",
+    "* Re-open a old run in order to fetch the metadata needed to reproduce it.\n",
+    "* Use fetched metadata to parametrize a new run with the same training loop.\n",
+    "\n",
+    "**This knowledge can be applied to any other scenario as well!**\n",
```

Comment:
Sounds a bit vague and not that useful. Maybe just leave it out? :)

Reply:
Indeed, I will remove it.